tests/generic_config_updater/test_eth_interface.py

```diff
@@ -294,7 +321,7 @@ def test_update_speed(duthost, ensure_dut_readiness):
 
         try:
             output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
-            if is_valid:
+            if is_valid and is_valid_speed_state_db(duthost, speed):
```

Comment:
>is_valid_speed_state_db [](http://example.com/codeflow?start=28&length=23)

Did you test one speed which is allowed by sonic yang model, but not valid in StateDB?

Reply:
Yes thank you, I updated the code to test this scenario